### DISCUSSION TITLE
How to properly move data from one system to another?

### DISCUSSION THREAD

**Discussion by apoorv569** [nextcloud/all-in-one#4853] [Questions] [answered]
I recently bought some new hardware (Processor, RAM, Drives etc..) and planning to build a `NAS` (using `TrueNAS`). ATM I have just 1 4TB drive that is being managed by `OpenMediaVault` (running in a `VM` on my `Proxmox` server), which has several different directories for share(s) for different `VM`s/Services, including `Nextcloud`.

What would be the best strategy to move my `Nextcloud` data from this 4TB drive to my new `TrueNAS` system?

This is how the directory structure looks like ATM in my `OpenMediaVault` server. 
![2024-06-20_16-21](https://github.com/nextcloud/all-in-one/assets/53279454/bc74023e-fff6-4311-a27e-8a3d50b5a39a)

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this question at https://help.nextcloud.com Thanks!

**Reply by apoorv569** [reply]
> Hi, please ask this question at https://help.nextcloud.com Thanks!

OK, thanks. I'm posting the link the forum post I created there in-case anyone in future wants to see it.
https://help.nextcloud.com/t/how-to-properly-move-data-from-one-system-to-another/195654
